Changes relative logo link with fully-qualified URL

naps62 opened the pull request:
Why:
- When copying this template to use in other projects, the relative link
  to Subvisual's logo will not work

This change addresses the need by:
- Making the link a fully-qualified URL instead, so that no change is
  needed on that line when using the template

zamith commented on code-review/README.md:
```diff
@@ -91,3 +91,4 @@ Ruby on Rails review
 * Review whether new scheduler (`cron`) tasks have been added and whether there
   is a related todo in the project management system to add it during the
   staging and production deploys.
+* Review whether there are new requirements being added to the production or staging machine. This includes new software packages, new secret keys in the app's environment, background services, network configurations, etc.
```
This is not indented according to the rest of the file.

Other than that, what should I do, if I find there are new things added?

naps62 replied:
fixed the indentation

If you see new things added, you might want to make sure that someone (either the author of the PR or the SysAdmin) already made the required changes to the server before merging this
I updated the text to include this as well

zamith replied:
Oh ok, then what about:

>  Review whether there are new dependencies that will require changes to the production or staging machine(s).

zamith commented on github/README.md:
```diff
@@ -0,0 +1,25 @@
+General usage
+-------------
+
+* Github is not only for developers. If you're a Designer or Project Manager or even a tester, you might be requested in a discussion
```
The tester could be capitalized as well.

naps62 replied:
fixed

naps62 commented:
@rmdmachado can you review this as well?

rmdmachado commented:
Looks good.

naps62 committed:
Changes relative logo link with fully-qualified URL

Why:

* When copying this template to use in other projects, the relative link
to Subvisual's logo will not work

This change addresses the need by:

* Making the link a fully-qualified URL instead, so that no change is
needed on that line when using the template
